service/alerter/service.go

```diff
@@ -227,3 +283,16 @@ func (s Service) UpdateOrphanResourceMetrics(resourceType string, resourceNames
 		s.logger.Log("info", fmt.Sprintf("orphan %s names are %s", resourceType, strings.Join(resourceNames, ",")))
 	}
 }
+
+// UpdateOrphanClusterMetrics updates the metric and logs the results.
+func (s Service) UpdateOrphanClusterMetrics(resourceType string, clusterIDs []string) {
+	resourceCount := len(clusterIDs)
+
+	orphanClustersTotal.WithLabelValues(resourceType).Set(float64(resourceCount))
+
+	s.logger.Log("info", fmt.Sprintf("alerter service found %d clusters with missing resources", resourceCount))
```

Comment:
Nitpick but I think the log message should say missing AWS resources. I think thats clearer and matches the metrics help text.

Reply:
Sure, done. I've also taken the opportunity to add an additional test case.